README.md

```diff
@@ -45,144 +52,86 @@ pip install eqlpy
 
 You can find the latest version on the [Python Package Index (PyPI)](https://pypi.org/project/eqlpy).
 
-## Usage
-
-### Importing the package
-
-```python
-from eqlpy import (
-    EqlInt,
-    EqlBool,
-    EqlDate,
-    EqlFloat,
-    EqlText,
-    EqlJsonb,
-    EqlRow
-)
-```
+TODO: How to install EQL itself?
 
-### EQL value classes
+## Usage with Django
 
-Each EQL value class inherits from the base `EqlValue` class and handles specific data types.
+### Defining an encrypted field
 
-#### EqlInt
-
-Handles integer values.
+Let's say you have a customer model with an encrypted `name` field.
+You can use the `EncryptedText` type when defining the field in the model:
 
 ```python
-eql_int = EqlInt(42, 'users', 'age')
-db_format = eql_int.to_db_format()
-print(db_format)
-```
+from django.db import models
+from eqlpy.eqldjango import *
 
-#### EqlBool
+class Customer(models.Model):
+    name = EncryptedText(table="customers", column="name")
+```
 
-Handles boolean values.
+The above customer model would create a database table like this:
 
-```python
-eql_bool = EqlBool(True, 'users', 'is_active')
-db_format = eql_bool.to_db_format()
-print(db_format)
+```sql
+CREATE TABLE myapp_customer (
+    "id" bigint NOT NULL PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    "name" cs_encrypted_v1 NOT NULL,
+);
 ```
 
-#### EqlDate
+[!NOTE]
+The column type in the database is `cs_encrypted_v1` which is encrypted type defined in [EQL](https://github.com/cipherstash/encrypt-query-language/blob/main/docs/reference/PAYLOAD.md). Internally EQL uses JSON.
 
-Handles date values.
+`eqlpy` supports many different data types, including numbers, dates and JSON.
+See the [Reference](#) docs for more information.
 
-```python
-from datetime import date
+TODO: Link to the reference docs.
 
-eql_date = EqlDate(date.today(), 'users', 'created_at')
-db_format = eql_date.to_db_format()
-print(db_format)
-```
 
-#### EqlFloat
+### Inserting data
 
-Handles floating-point values.
+`eqlpy` automatically encrypts values when saving to the database.
+You can interact with the Django model in the same way as you normally would.
 
-```python
-eql_float = EqlFloat(3.14, 'measurements', 'value')
-db_format = eql_float.to_db_format()
-print(db_format)
+```py
+>>> customer = Customer(name="Fred Flintstone")
+>>> customer.save()
 ```
 
-#### EqlText
-
-Handles text values.
+When retrieving a record, `eqlpy` automatically decrypts the value using the credentials defined in the setup step.
 
-```python
-eql_text = EqlText('Hello, World!', 'messages', 'content')
-db_format = eql_text.to_db_format()
-print(db_format)
+```py
+>>> customer = Customer.objects.get(customer.id)
+>>> customer.name
+'Fred Flintstone'
 ```
 
-#### EqlJsonb
+TODO: Show what is contained in the database at this stage.
```

Comment:
```suggestion
This record looks like this when accessed on Proxy:

# select * from customers;
 id  | is_citizen | start_date | weight | age |                                       name                                       | extra_info | visit_count
 -----+------------+------------+--------+-----+----------------------------------------------------------------------------------+------------+-------------
  446 |            |            |        |     | {"k":"pt","p":"Fred Flintstone","i":{"t":"customers","c":"name"},"v":1,"q":null} |            |
  (1 row)

and directly on PostgreSQL:

 id  | is_citizen | start_date | weight | age |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         name                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | extra_info | visit_count
-----+------------+------------+--------+-----+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------+-------------
 446 |            |            |        |     | {"c": "mBbL^r9sYr{**CKA_6C{IDN*%A*Hh3Y^S6yIkF*Ez{m&ib*gwg#*A)&{F6mjUwBj_aVx|i6EsZUQXbrN)(~iOu-@!cWr+e^P~2NpGTB?trC@Gwq;6qtWxx", "i": {"c": "name", "t": "customers"}, "k": "ct", "m": [77, 1666, 179, 786, 1429, 976, 2005, 1696, 182, 1317, 405, 1850, 1337, 768, 238, 253, 460, 443, 270, 779, 291, 1008, 1051, 1835, 1860, 1665, 538, 1067, 1914, 793, 272, 591, 684, 1861, 2037, 318, 525, 155, 672, 328, 1035, 853, 388, 1721, 1547, 1705, 613, 852, 1947, 1276, 481, 273, 1494, 1378, 1599, 1434, 319, 1794, 816, 1960, 379, 149, 1131, 1309, 204, 438, 1989, 1718, 29, 1413, 1965, 738, 1324, 2006, 802, 201, 251], "o": "(\"{\"\"(\\\\\"\"\\\\\\\\\\\\\\\\xe3596bf64b97d034fe0d967c6b224819467f41057f2fd61e8f2abb873e2c26b5e937781acd24165f09d73e810d516979bc295f66fa1aae86a457ce97d04e42680bb7160ae1f742ec865c06c1cafe6ad87c41bfdb6cda491e83061ca86f7a523c4edc033132370e51e5971bf3000c2a29a526f49d1323a224dfc782667d741a53c9b88936bef7e0fafb5b7d55cf9fdeb7e567a31ed53bde5b628b7f906421126f29bb5195e88fe8df669d5002bd2ed31db5cd8afe560d7dcbf5400c6e4b9eeba94a00078ee2ca5e138c37735648d2248c483156899c560a52fe7f1e41b6146d5382630bf2895dd0438d4de7f75c0c5ecb5f78f04736fff266a21713f701758c32b3dc8e17e01ea1406ad95c6c5cc950140f22e679b12d799b7a56890ded70188ab5c3a965d88bb9f9bb76d0d5f73746a5a048cbb78a8c18bdc5ebadf689ce881c566811fbf86243dd26ab6764656b86ad14358cf973ae99746ad24d7539362e69289a7de27920366bc619663cf5f7e401a6423eaff8699b95eccaf53989d7dd91af08f574932c1b79d89c95973d4dc45e0cfc3fdd0c7bf7dd\\\\\"\")\"\", \"\"(\\\\\"\"\\\\\\\\\\\\\\\\xe969121212121212714451b0a9ac6cd7de810a31ee27792a608ff20fb9d707c0d8de75fa2dd800f7b719322d123b4b314e9e08405426fcbb9ae8fb7e5a7ba009f4fa6f8f31d5e6b61fce62db66d1719390d6eae6aa60d89509de194056631376bad1c56db834982cd02cf69aaf7a8dfddeaab23f3ccf7fb0875d4ac3199188ea30b4d508f1e95810ac1274747298cb8afe20c6c8029811106392ce71683068c1b13c69474c9c81ec4a52b0631722d727bc02a801130369b73b38b4a1acf53cbc16c5773995a788fbc8d17f02945a3a95970aeaf4c08675adf13d29e5bca8e5ea09329e9fd41694a0af963c9a4158cac86257d9adbb66c4fa41b1878bec83394369545270e816a5212d6d5b7f18965c181040a3fcddf1b11d0ff90a5812840eaa8c9768705abae4de1749d9c7a2c608e4e7c9191b2501397e8b8566ae9f0b6ca82496ead7418fdc9d4ec243c8b941a67fae006c64022ada7e6679bffa4d557897ebfb488f795fd78032ff91de8cd186f1721ebf911beeac1810effabd0129a70199a609d79c5fbd8f6d24b3e7304c97358e7454d386f7c0f7\\\\\"\")\"\"}\")", "u": "bc9295f8303391f437f07b9298335347cc17a53a52025457456606e98955f7b8", "v": 1} |            |

```

Not sure if you want the PostgreSQL version, but added it anyway. Also, I don't think I can use triple backticks inside suggestions. So my suggestion is either copy or commit this, then edit as required if it helps...